deploy/bin/build-local-edge-endpoint-image.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
```

Comment:
nit: is this script intended to be executed manually (in addition to during cicd?). If so it might be useful to add a little description

Reply:
Yes it is. Good call. 

More generally, I need to write a whole doc on how to do development. I didn't want to block this PR on that doc.

Reply:
Yeah i agree that doc would be helpful in the future